src/sidebar/services/frame-sync.js

```diff
@@ -347,7 +357,7 @@ export class FrameSyncService {
    */
   focusAnnotations(tags) {
     this._store.focusAnnotations(tags);
-    this._guestRPC.call('focusAnnotations', tags);
+    this._guestRPC.forEach(rpc => rpc.call('focusAnnotations', tags));
```

Comment:
In future it might be possible to be smarter about only sending these events to the right guest. However it isn't that important because guests ignore these messages if they are not displaying a particular annotation.